projects/v1/wikipedia.wmf.yaml

```diff
@@ -90,6 +90,10 @@ paths:
       - path: v1/random.yaml
         options: '{{merge({"random_cache_control": "s-maxage=2, max-age=1"},
                       options.mobileapps)}}'
+      - path: v1/talk.yaml
+        options:
+          response_cache_control: '{{options.purged_cache_control}}'
```

Comment:
But it's not purged.. The endpoints we purge are cached quite extensively, so this will draw varnish responses very much out of date. I'm not really sure what the correct value would be for talk pages, it's more a product decision.

Reply:
Yes, we still need purging. Not sure how long we should cache this. Probably don't need to keep around very long. I suspect this to be low traffic, at least initially, since initially this will be used by the iOS app to display a user talk pages.

Reply:
For the purging we need to mangle change-prop then. Could you create a ticket so that we don't forget. I can make a rule.

Reply:
Here's a strawman proposal: How about 5 minutes?

Reply:
> For the purging we need to mangle change-prop then. Could you create a ticket so that we don't forget. I can make a rule.

https://phabricator.wikimedia.org/T226669

Reply:
> [...] Could you create a ticket so that we don't forget. [...]
https://phabricator.wikimedia.org/T226670